docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm

```diff
@@ -203,7 +203,7 @@ <h3>Summary</h3>
 				</tr>
 				<tr>
 					<td>
-						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">getActiveSectionIndex</a>() </pre>
+						<pre class="intable prettyprint"><a href="#method-getActiveSectionIndex">number getActiveSectionIndex</a>() </pre>
```

Comment:
Just a question.
Is it OK to use "number", not "Number"?

Reply:
> Just a question.
> Is it OK to use "number", not "Number"?

Properly lower case only for simple types:
- number
- string

Rest of types should be from capital letter like:
- Object
- Event
- HTMLElement

Reply:
@dorlinski 
Would you change "Number" to "number"? Thank you.

Reply:
Yes, but maybe I will do it in separate PR, in which I correct all?

Reply:
Do I have to change all types to lowercase (Boolean -> boolean, Number -> number, String -> string)? And maybe it better to use upper case everywhere?

Reply:
Done